crates/ide_db/src/tests/sourcegen_lints.rs

```diff
@@ -18,6 +18,7 @@ fn sourcegen_lint_completions() {
 
     let mut contents = String::from(
         r"
+#[derive(Clone)]
```

Comment:
I'm added this for clone Lint.


The difference of `crates/ide_db/src/helpers/generated_lints.rs` is due to the application of this